srml/contract/src/wasm/runtime.rs

```diff
@@ -372,6 +372,8 @@ define_env!(Env, <E: Ext>,
 		ctx,
 		init_code_ptr: u32,
 		init_code_len: u32,
+		rent_allowance_ptr: u32,
+		rent_allowance_len: u32,
```

Comment:
Hm, good point, tbh I didn't think too much about providing this as an argument. 

I am still a bit inclined that `rent_allowance` should be set only by a contract: in the end, the contract itself decides how it would like to control its funds.

Reply:
ok I remove the argument and provide ext_(set_)rent_allowance